brainglobe_atlasapi/atlas_generation/atlas_scripts/osten_mouse.py

```diff
@@ -21,7 +20,7 @@
 from brainglobe_atlasapi.config import DEFAULT_WORKDIR
 from brainglobe_atlasapi.structure_tree_util import get_structures_tree
 
-PARALLEL = False  # disable parallel mesh extraction for easier debugging
+# disable parallel mesh extraction for easier debugging
```

Comment:
```suggestion
```
also remove further occurrences of this comment, which is now out of date (and wasn't that useful in the first place 😁 )